docs/source/getting_started.rst

```diff
@@ -4,89 +4,89 @@ Getting Started
+the `Amazon App Store <http://www.amazon.com/Martin-Denham-And-Bible/dp/B004Z2KKYK>`_,
+and directly from the `Releases section of our GitHub <https://github.com/AndBible/and-bible/releases/latest>`_
+source code repository.
+
+.. image:: /images/google_play_badge.png
+   :height: 50
    :target: https://play.google.com/store/apps/details?id=net.bible.android.activity
 
-Click the image above to get the app from the Google Play store.
 
-F-Droid
--------
-
-.. image:: /images/f-droid-badge.png
-   :width: 200
+.. image:: /images/f-droid_badge.png
+   :height: 50
    :target: https://f-droid.org/packages/net.bible.android.activity/
 
-Click the image above to get the app from F-Droid.
-
-Amazon App Store
-----------------
-
-.. image:: /images/amazon-badge.png
-   :width: 200
+.. image:: /images/amazon_badge.png
+   :height: 50
    :target: http://www.amazon.com/Martin-Denham-And-Bible/dp/B004Z2KKYK
 
-Click the image above to get the app from the Amazon App Store.
-
-GitHub Releases
----------------
-
-.. image:: /images/obtainium-badge.png
-   :width: 200
+.. image:: /images/obtainium_badge.png
+   :height: 50
    :target: https://obtainium.imranr.dev/
 
-Click the image above to get Obtainium which can download (and update)
-the apk from Github. Alternatively, you can download and install the
-apk directly from the "Assets" sub-menu in the
-`GitHub Releases <https://github.com/AndBible/and-bible/releases/latest>`_
-
-(Note: An apk downloaded directly from Github will not automatically update)
+Note: An apk downloaded directly from the "Assets" sub-menu of the Github Releases
+will not automatically update. You can use a tool like `Obtainium <https://obtainium.imranr.dev/>`_
+to download (and update) the apk from Github.
 
-Windows 11 setup
------------------
+Linux setup
+-----------
 
-** Note: Windows Subsystem for Android has been discontinued as of March 5th, 2025.
+#. `Install Waydroid <https://docs.waydro.id/usage/install-on-desktops>`_
+#. Run Waydroid and install GAPPS version of Android
+   (The GAPPS version is required to update WebView and for Google Drive sync).
+#. `Set up Play Services <https://docs.waydro.id/faq/google-play-certification>`_.
+#. `Disable the on-screen keyboard <https://docs.waydro.id/faq/disable-on-screen-keyboard>`_.
+#. Install / Upgrade `WebView <https://play.google.com/store/apps/details?id=com.google.android.webview>`_
+   from Google Play. This fixes some issues.
+#. Install `Google Speech Services <https://play.google.com/store/apps/details?id=com.google.android.tts>`_
+   from Google Play (if you want to use Text to Speech).
+#. Install `Google Drive <https://play.google.com/store/apps/details?id=com.google.android.apps.docs>`_
+   from Google Play (not needed for Google Drive sync, but you might need it if
+   you have stored any document backups etc in Google Drive).
+#. Install AndBible with your preferred method (See: `Installation`_).
```

Comment:
Can you also modify WIKI to have just a link there to docs if you move stuff from there?

Reply:
Yes, you could remove duplicate information from there and leave the link there.